addons/binding/org.openhab.binding.netatmo/ESH-INF/config/config.xml

```diff
@@ -83,6 +83,13 @@
 			<description>Id of the Device (mac address)</description>
 			<required>true</required>
 		</parameter>
+
+		<parameter name="refreshInterval" type="integer" required="false">
+			<label>Refresh Interval</label>
+			<description>The refresh interval to poll Netatmo API (in ms).</description>
```

Comment:
Please make this seconds, I see no reason to specify this in ms,l maybe even minutes would make sense? Or something with a unit?

Reply:
Yes, obviously this should be seconds but I did not want to introduce a breaking change in configuration for no good reasons. This is like this since a while.

Reply:
Sorry, you're right. I introduced this. But as we've always talked about refresh period in ms in this binding, I thought it would be more consistent.

Reply:
But imho not from the user perspective so under the hood ms are fine but let's not confront end users with it :-)

Reply:
sure, but it means more deeper refactoring as this is based on ancestor class. So I'll introduce a breaking change in current configurations. WDYT ?